packages/common/tsconfig.json

```diff
@@ -1,22 +1,22 @@
-{
-    "compilerOptions": {
-      "target": "es5",
-      "module": "commonjs",
-      "lib": ["es2017", "es7", "es6", "dom"],
-      "declaration": true,
-      "outDir": "features",
-      "strict": false,
-      "esModuleInterop": true,
-      "experimentalDecorators":true
-    },
-    "include": [
-      "./src"
-    ],
-    "exclude": [
-      "node_modules",
-      "**/*.spec.ts",
-      "AppData",
-      "../mobile/*.ts",
-      "../mobile/*.ts"
-    ]
+{
+    "compilerOptions": {
+      "target": "es5",
+      "module": "commonjs",
+      "lib": ["es2017", "es7", "es6", "dom"],
+      "declaration": true,
+      "outDir": "features",
+      "strict": false,
+      "esModuleInterop": true,
+      "experimentalDecorators":true
+    },
+    "include": [
+      "./features"
```

Comment:
czemu fetures? i czemu usunołeś wszystkow w  /src ?

Reply:
chyba wywalę to i zacznę z mastera